stock_picking_procure_method/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,37 @@
+This is an example scenario with two warehouses. WH2 will be allways supplied
+through WH1.
+
+Common steps to `mrp` and `purchase` procurements:
+
+#. Go to *Inventory > Configuration > Settings > Warehouse* and set *Multi-Step
+   Routes* on.
+#. Go to *Inventory > Configuration > Warehouse Management > Warehouses*
+#. Create **WH1** with either *Manufacture to Resupply* or *Buy to
+   Resupply* this warehouse or both set.
+#. Create **WH2** setting off *Manufacture to Resupply* and *Buy to
+   Resupply*. Set **WH1** as the *Resupply Warehouse*.
+#. Go to *Inventory > Configuration > Warehouse Management > Routes* and click
+   on the *Manufacture* one.
```

Comment:
This is still marking `Manufacture`.

Reply:
Fixed

Reply:
But you have to indicate that it should be first unarchived.